src/main/java/teammates/ui/controller/XlsFileDownloadResult.java

```diff
@@ -44,7 +44,8 @@ private XSSFWorkbook getWorkBook() {
         int rowNumber = 0;
         String[] lines = fileContent.split(Const.EOL);
         for (String line : lines) {
-            String[] str = line.split(",");
+            // Split by comma only if comma has a zero or even number of quotes in front of it.
+            String[] str = line.split(",(?=([^\"]*\"[^\"]*\")*[^\"]*$)", -1);
```

Comment:
We should unit test regexes as they are sources of subtle errors.  Define it as a constant and then use the same constant in a unit test. I think there are examples of regex unit tests somewhere.